torch/nn/_functions/dropout.py

```diff
@@ -58,8 +58,11 @@ def backward(ctx, grad_output):
 class FeatureDropout(Dropout):
 
     @staticmethod
-    def symbolic(input, p=0.5, train=False, inplace=False):
-        return None
+    def symbolic(g, input, p=0.5, train=False, inplace=False):
+        # NB: No FeatureDropout in ONNX, use Dropout instead.
+        if train:
+            raise ValueError("In train mode, FeatureDropout is different from Dropout.")
```

Comment:
Imagine you get this error when you export a model. You know it’s different, that’s why you used it instead. This error doesn’t help the user figure out what went wrong at all

Reply:
Yes, people may export the model in train mode and use it in inference mode, and vise versa. The message can still warn users when training flag is set true during exporting.

And this is just a temporary solution.
@ezyang has requested the FeatureDropout in https://github.com/onnx/onnx/issues/113.
We will update the exporter as long as it is introduced into ONNX.

Reply:
I think @apaszke meant that it would be better to change the error message to something like
```
onnx export for FeatureDropout not supported in train mode.
```

Reply:
Yes, that's exactly what I meant. People never use FeatureDropout directly, and telling that export failed because "FeatureDropout != Dropout" doesn't really carry any usable information for them

Reply:
@fmassa Thanks for the explanation. @apaszke Make sense. I will update the error message.